pop_tools/grid.py

```diff
@@ -391,3 +391,128 @@ def _compute_corners(ULAT, ULONG):
     corner_lon[0, :, 3] = corner_lon[1, :, 3] - (corner_lon[2, :, 3] - corner_lon[1, :, 3])
 
     return corner_lat, corner_lon
+
+
+@jit(cache=True, nopython=True)
```

Comment:
How about enabling numba to automatically parallelize (and hopefully perform other optimization) this function?

```suggestion
@jit(cache=True, nopython=True, parallel=True)
```